emop-maven-plugin/src/main/java/edu/cornell/RppHandlerMojo.java

```diff
@@ -0,0 +1,166 @@
+package edu.cornell;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.Sets;
+import edu.cornell.emop.maven.AgentLoader;
+import edu.cornell.emop.util.Util;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+@Mojo(name = "rpp-handler", requiresDirectInvocation = true, requiresDependencyResolution = ResolutionScope.TEST)
+public class RppHandlerMojo extends MonitorMojo {
```

Comment:
Won't this cause MonitorMojo to execute before RppHandlerMojo? At the moment MonitorMojo is configured entirely for RPS at this seems like it would add a lot of extra overhead at runtime that we don't need

Reply:
So I created a change in the `lifecycle.xml` file that stipulates that for the `rpp` lifecycle, the `RppHandlerMojo` will be directly invoked. So I think we won't have any problems where `MonitorMojo` will be invoked before `RppHandlerMojo`.

Reply:
Done!